docs/integrations/connector-support-levels.md

```diff
@@ -6,54 +6,54 @@ products: all
 | **Support: Self-Managed Enterprise** | Supported\*                               | No Support                                                                                             | Supported\*\*                                                                                                                                                                                                                                                      |
 | **Support: Community (OSS)**         | Slack Support only                        | No Support                                                                                             | Slack Support only                                                                                                                                                                                                                                                 |
 
-\*For Certified connectors, Official Support SLAs are only available to customers with Premium
+\*For Integrations, Official Support SLAs are only available to customers with Premium
 Support included in their contract. Otherwise, please use our support portal and we will address
 your issues as soon as possible.
 
 \*\*For Custom connectors, Official Support SLAs are only available to customers with Premium
 Support included in their contract. This support is provided with best efforts, and
 maintenance/upgrades are owned by the customer.
 
-## Certified
+## Integrations
 
-A **Certified** connector is actively maintained and supported by the Airbyte team and maintains a
+An **Integration** type connector is actively maintained and supported by the Airbyte team and maintains a
 high quality bar. It is production ready.
 
-### What you should know about Certified connectors:
+### What you should know about Integrations:
 
-- Certified connectors are available to all users.
-- These connectors have been tested and vetted in order to be certified and are production ready.
-- Certified connectors should go through minimal breaking change but in the event an upgrade is
+- Integrations are official Airbyte connectors that are available to all users.
+- These connectors have been tested and vetted. They are production ready.
+- Integrations should go through minimal breaking change but in the event an upgrade is
   needed users will be given an adequate upgrade window.
 
-## Community
+## Marketplace
 
-A **Community** connector is maintained by the Airbyte community until it becomes Certified. Airbyte
+A **Marketplace** connector is maintained by the Airbyte community until it becomes an Integration. Airbyte
```

Comment:
```suggestion
A **Marketplace** connector is maintained by the community members until it becomes an official Integration. Airbyte
```

Reply:
To put more emphasis on community and less on Airbyte. Set the right support level expectation.